apps/api/src/app/app.module.ts

```diff
@@ -16,14 +19,13 @@ import { PosthogModule } from 'nestjs-posthog';
       isGlobal: true,
       envFilePath: ['.env.local', '.env'],
     }),
-    RedisModule.forRootAsync({
-      useFactory: (config: ConfigService) => {
-        return {
-          name: 'db',
-          url: config.get('REDIS_URI'),
-        };
-      },
-      inject: [ConfigService],
+    RedisModule.forRoot({
+      readyLog: true,
+      config: {
+          host: 'localhost',
```

Comment:
Hey @suyashgautam , can you check the previous dependency injection method used. Its not good practise to expose env variables here. We need to fetch it from .env file.

Reply:
Hey @radhay-samagra , I have fixed the hard coded env variables issue. Do let me know if there is anything more I can improve.